conf/machine/imx23evk.conf

```diff
@@ -20,4 +20,4 @@ UBOOT_MACHINE = "mx23evk_config"
 
 KERNEL_DEVICETREE = "imx23-evk.dtb"
 
-MACHINE_FEATURES = "usbgadget usbhost vfat touchscreen"
+MACHINE_FEATURES = "usbgadget usbhost vfat touchscreen screen"
```

Comment:
Could you please remove the override and switch to using the `imx-base` one? Thank you.

Reply:
Since 1f8df81eaa8c22329d99a473c45b3c368a363c31, the difference is that imx23evk no longer has `alsa`; apparently it wasn't ready at the time. I don't know about the state of imx23evk audio support, is it okay to add it back now?

Reply:
I think it makes sense.

Reply:
Done